packages/experimental/RadioGroup/src/Radio/Radio.styling.ts

```diff
@@ -16,9 +16,10 @@ export const stylingSettings: UseStylingOptions<RadioProps, RadioSlotProps, Radi
           display: 'flex',
           alignItems: tokens.alignItems,
           flexDirection: tokens.flexDirection,
-          minHeight: 20,
-          marginTop: 0,
-          marginRight: globalTokens.spacing.s,
+          // minHeight: 20,
+          // marginTop: 0,
+          // marginRight: globalTokens.spacing.s,
```

Comment:
Is it needed?

Reply:
Nope, accidentally left that in. Thanks for catching that!